OSDOCS-7075: Documented support for multiple subnets (#75017)
openshift/openshift-docs: modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

If you have established different network segments or subnets for remote worker nodes as described in the section on "Establishing communication between subnets", you must specify the subnets in the `machineNetwork` configuration setting if the workers are using static IP addresses, bonds or other advanced networking. When setting the node IP address in the `networkConfig` parameter for each remote worker node, you must also specify the gateway and the DNS server for the subnet containing the control plane nodes when using static IP addresses. This ensures the remote worker nodes can reach the subnet containing the control plane nodes and that they can receive network traffic from the control plane.

[IMPORTANT]
[NOTE]
====

> All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.

Author: Thanks, Ben. Should I remove:

[NOTE]
====
Deploying a cluster with multiple subnets requires using virtual media, such as `redfish-virtualmedia` and `idrac-virtualmedia`.
====

We do not mention multiple subnets in this section.

[IMPORTANT]
====

> You can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.

Reviewer: Have you considered using a snippet for this, since it is repeated?

Author: Yeah, good point. I'll do this!

> You can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.

Reviewer: Just a nit that might improve some already good writing
```suggestion
You can run control plane nodes in the same subnet or multiple subnets. If you run a multiple subnet environment, you can reduce the risk of your {product-title} cluster from failing due to a hardware failure or a network outage.
```

> For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Ensure that you configure an external load balancer in place of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".

Reviewer: Nit/suggestion to make that second sentence a little shorter.
```suggestion
For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Configure an external load balancer instead of the default load balancer by specifying `UserManaged` in the `loadBalancer.type` parameter of the `install-config.yaml` file. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```

[IMPORTANT]
====
You can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.

> For a multiple subnet environment, create a manifest configuration file and specify the external load balancer address in the `ingressVIPs` and `apiVIPs` parameters of the file. Ensure that you configure an external load balancer in place of the default load balancer. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".

Reviewer 2: Can we explicitly name the install-config field that needs to be set for user-managed loadbalancers? Looks like that would be loadBalancerType=UserManaged under the platform config options. Otherwise lgtm now.

> For a multiple subnet environment, create a manifest configuration file and specify more than one virtual IP address in the `ingressVIP` parameter of the file. You must also configure an external load balancer in place of the default load balancer. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".

Reviewer 2: Multiple ingress VIPs are not necessary here. The ingressVIPs parameter (note that the singular field name is deprecated) should just be the external loadbalancer address. Likewise for apiVIPs.

Author: Thanks. Ah, yes, I see this stated elsewhere that those parameters are deprecated. Updated.
